bootstrap-extensions/src/main/java/de/agilecoders/wicket/extensions/markup/html/bootstrap/form/datetime/DatetimePickerBehavior.java

```diff
@@ -59,8 +59,8 @@ public void renderHead(Component component, final IHeaderResponse response) {
     @Override
     public void onComponentTag(Component component, ComponentTag tag) {
         super.onComponentTag(component, tag);
-        Attributes.set(tag, "data-bs-target", "#" + component.getMarkupId());
-        Attributes.set(tag, "data-bs-toggle", "datetimepicker");
+        Attributes.set(tag, "data-target", "#" + component.getMarkupId());
```

Comment:
Is this correct ?
I thought BS 5.x now uses `data-bs-**` everywhere ?!
Or maybe the datetimepicker is not yet migrated to BS 5.x ? In that case let's set both `data-target` and `data-bs-target`. Hopefully this will make it future-proof.